### PR TITLE
Features-Password reset page custom redirect

### PR DESCRIPTION
**Pull request by ram35hi**

### CODE REVIEW

File: page-templates/user/password-reset.ejs [hunk 1]
```diff
@@ -91,6 +91,20 @@
                         </div>
                     </div>
                 </center>
+                <center>
+                    <div id="success-message-redirect" class="solo-horizontal-center" style="background-color: white;padding: 10px;">
+
```
Comment (nawazdhandala) [inline page-templates/user/password-reset.ejs]: Remove blank line.

File: page-templates/user/password-reset.ejs [hunk 2]
```diff
@@ -288,6 +302,7 @@
         $("#authenticate-error").hide();
         $("#authenticate-spinner").hide();
         $("#success-message").hide();
+        $("#success-message-redirect").hide();
         //$("#page-not-found-message").hide();
```
Comment (nawazdhandala) [inline page-templates/user/password-reset.ejs]: Uncomment this. If this line of code is not required. Remove it.

File: page-templates/user/password-reset.ejs [hunk 3]
```diff
@@ -370,8 +385,19 @@
...
+                        if(_authSettings.resetPasswordEmail.redirectURL.enabled===true){
+                            if (!_authSettings.resetPasswordEmail.redirectURL.URL) {
+                                 throwAuthError("Please add your app's password reset redirect URL in your CloudBoost App Settings.");
+                               }
+                            else{
+                                $("#success-message-redirect").show();
+                                setTimeout("location.href = _authSettings.resetPasswordEmail.redirectURL.URL;",2000);
+                            }
+                        }
+                         else{
```
Comment (nawazdhandala) [inline page-templates/user/password-reset.ejs]: Indent this properly.

**Comment by nawazdhandala** [comment]
Please check review and fix. Send PR after done.

**Comment by ram35hi** [comment]
Ok..I'll go on to it.

**Pull request merged**